oak_enclave_shim/src/main.rs

```diff
@@ -30,19 +30,26 @@ use oak_linux_boot_params::BootParams;
 //    try to execute it ourselves or try to parse it as an ELF file.
 // (A more clever thing would be to embed an ELF file here that causes an ud2...)
 #[naked]
+#[no_mangle]
 #[link_section = ".payload"]
 extern "C" fn payload() -> ! {
     unsafe {
         asm!("ud2", options(noreturn));
     }
 }
 
+extern "C" {
+    #[link_name = "payload_start"]
```

Comment:
Is payload what you refer to as application in the README?

Reply:
Yes.